test/test.js

```diff
@@ -119,4 +119,8 @@ describe('postcss-increase-specificity', function() {
 	it('should not change the descendant rules of @keyframes', function() {
 		return testPlugin('./test/fixtures/keyframes.css', './test/fixtures/keyframes.expected.css');
 	});
+
+	it('should support disabled blocks of rules', function() {
+		return testPlugin('./test/fixtures/disabled-block.css', './test/fixtures/disabled-block.expected.css');
+	});
```

Comment:
Let's add another test for multiple `disable/enable`

```css
/* postcss-increase-specificity disable */
.noop {
	background: #f00;
}
/* postcss-increase-specificity enable */

.bar {
  color: #ef0;
}

/* postcss-increase-specificity disable */
.noop2 {
	background: #f00;
}
/* postcss-increase-specificity enable */
```

---

Let's add a test for just a `disable` comment on it's own

```css
/* postcss-increase-specificity disable */
.foo {
	background: #f00;
}

.bar {
  color: #ef0;
}

/* some other random comment that should not enable */
.baz {
  color: #040;
}
```

And another one in this same vein

```css
.foo {
	background: #f00;
}

/* postcss-increase-specificity disable */
.bar {
  color: #ef0;
}

.baz {
  color: #040;
}
```

Reply:
Done!